--- ./usr/include/stdio_ext.h.orig	2023-10-19 22:30:05.211235393 +0700
+++ ./usr/include/stdio_ext.h	2023-10-19 22:34:20.456253879 +0700
@@ -42,20 +42,15 @@
  * [__fbufsize(3)](http://man7.org/linux/man-pages/man3/__fbufsize.3.html) returns the size of
  * the stream's buffer.
  *
- * Available since API level 23.
  */
-
-#if __ANDROID_API__ >= 23
-size_t __fbufsize(FILE* __fp) __INTRODUCED_IN(23);
+size_t __fbufsize(FILE* __fp) __INTRODUCED_IN(21);
 
 /**
  * [__freadable(3)](http://man7.org/linux/man-pages/man3/__freadable.3.html) returns non-zero if
  * the stream allows reading, 0 otherwise.
  *
- * Available since API level 23.
  */
-int __freadable(FILE* __fp) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
+int __freadable(FILE* __fp) __INTRODUCED_IN(21);
 
 
 /**
@@ -74,13 +69,9 @@
  * [__fwritable(3)](http://man7.org/linux/man-pages/man3/__fwritable.3.html) returns non-zero if
  * the stream allows writing, 0 otherwise.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int __fwritable(FILE* __fp) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int __fwritable(FILE* __fp) __INTRODUCED_IN(21);
 
 /**
  * [__fwriting(3)](http://man7.org/linux/man-pages/man3/__fwriting.3.html) returns non-zero if
@@ -98,36 +89,30 @@
  * [__flbf(3)](http://man7.org/linux/man-pages/man3/__flbf.3.html) returns non-zero if
  * the stream is line-buffered, 0 otherwise.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int __flbf(FILE* __fp) __INTRODUCED_IN(23);
+int __flbf(FILE* __fp) __INTRODUCED_IN(21);
 
 /**
  * [__fpurge(3)](http://man7.org/linux/man-pages/man3/__fpurge.3.html) discards the contents of
  * the stream's buffer.
  *
- * Available since API level 23.
  */
-void __fpurge(FILE* __fp) __INTRODUCED_IN(23);
+void __fpurge(FILE* __fp) __INTRODUCED_IN(21);
 
 /**
  * [__fpending(3)](http://man7.org/linux/man-pages/man3/__fpending.3.html) returns the number of
  * bytes in the output buffer.
  *
- * Available since API level 23.
  */
-size_t __fpending(FILE* __fp) __INTRODUCED_IN(23);
+size_t __fpending(FILE* __fp) __INTRODUCED_IN(21);
 
 /**
  * [_flushlbf(3)](http://man7.org/linux/man-pages/man3/_flushlbf.3.html) flushes all
  * line-buffered streams.
  *
- * Available since API level 23.
  */
-void _flushlbf(void) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
+void _flushlbf(void) __INTRODUCED_IN(21);
 
 
 /**
@@ -155,12 +140,8 @@
  *
  * Returns the current locking style, `FSETLOCKING_INTERNAL` or `FSETLOCKING_BYCALLER`.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int __fsetlocking(FILE* __fp, int __type) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int __fsetlocking(FILE* __fp, int __type) __INTRODUCED_IN(21);
 
 __END_DECLS
