--- ./usr/include/error.h.orig	2023-10-20 17:39:29.047229959 +0700
+++ ./usr/include/error.h	2023-10-20 17:40:45.986235532 +0700
@@ -45,8 +45,7 @@
  * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-extern void (* _Nullable error_print_progname)(void) __INTRODUCED_IN(23);
+extern void (* _Nullable error_print_progname)(void);
 
 /**
  * [error_message_count(3)](http://man7.org/linux/man-pages/man3/error_message_count.3.html) is
@@ -54,7 +53,7 @@
  *
  * Available since API level 23.
  */
-extern unsigned int error_message_count __INTRODUCED_IN(23);
+extern unsigned int error_message_count;
 
 /**
  * [error_one_per_line(3)](http://man7.org/linux/man-pages/man3/error_one_per_line.3.html) is
@@ -63,7 +62,7 @@
  *
  * Available since API level 23.
  */
-extern int error_one_per_line __INTRODUCED_IN(23);
+extern int error_one_per_line;
 
 /**
  * [error(3)](http://man7.org/linux/man-pages/man3/error.3.html) formats the given printf()-like
@@ -72,7 +71,7 @@
  *
  * Available since API level 23.
  */
-void error(int __status, int __errno, const char* _Nonnull __fmt, ...) __printflike(3, 4) __INTRODUCED_IN(23);
+void error(int __status, int __errno, const char* _Nonnull __fmt, ...) __printflike(3, 4);
 
 /**
  * [error_at_line(3)](http://man7.org/linux/man-pages/man3/error_at_line.3.html) formats the given
@@ -82,8 +81,6 @@
  *
  * Available since API level 23.
  */
-void error_at_line(int __status, int __errno, const char* _Nonnull __filename, unsigned int __line_number, const char* _Nonnull __fmt, ...) __printflike(5, 6) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+void error_at_line(int __status, int __errno, const char* _Nonnull __filename, unsigned int __line_number, const char* _Nonnull __fmt, ...) __printflike(5, 6);
 
 __END_DECLS
