--- ./usr/include/stdio_ext.h.orig	2023-12-02 19:10:31.937359112 -0500
+++ ./usr/include/stdio_ext.h	2023-12-02 19:31:11.050189053 -0500
@@ -42,20 +42,16 @@
  * [__fbufsize(3)](http://man7.org/linux/man-pages/man3/__fbufsize.3.html) returns the size of
  * the stream's buffer.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-size_t __fbufsize(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
+size_t __fbufsize(FILE* _Nonnull __fp);
 
 /**
  * [__freadable(3)](http://man7.org/linux/man-pages/man3/__freadable.3.html) returns non-zero if
  * the stream allows reading, 0 otherwise.
  *
- * Available since API level 23.
  */
-int __freadable(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
+int __freadable(FILE* _Nonnull __fp);
 
 
 /**
@@ -74,12 +70,9 @@
  * [__fwritable(3)](http://man7.org/linux/man-pages/man3/__fwritable.3.html) returns non-zero if
  * the stream allows writing, 0 otherwise.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
 int __fwritable(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
 
 
 /**
@@ -101,25 +94,21 @@
  * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int __flbf(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
+int __flbf(FILE* _Nonnull __fp);
 
 /**
  * [__fpurge(3)](http://man7.org/linux/man-pages/man3/__fpurge.3.html) discards the contents of
  * the stream's buffer.
  *
- * Available since API level 23.
  */
-void __fpurge(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
+void __fpurge(FILE* _Nonnull __fp);
 
 /**
  * [__fpending(3)](http://man7.org/linux/man-pages/man3/__fpending.3.html) returns the number of
  * bytes in the output buffer. See __freadahead() for the input buffer.
  *
- * Available since API level 23.
  */
-size_t __fpending(FILE* _Nonnull __fp) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
+size_t __fpending(FILE* _Nonnull __fp);
 
 
 /**
@@ -138,12 +127,9 @@
  * [_flushlbf(3)](http://man7.org/linux/man-pages/man3/_flushlbf.3.html) flushes all
  * line-buffered streams.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-void _flushlbf(void) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
+void _flushlbf(void);
 
 
 /**
@@ -171,12 +157,8 @@
  *
  * Returns the current locking style, `FSETLOCKING_INTERNAL` or `FSETLOCKING_BYCALLER`.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int __fsetlocking(FILE* _Nonnull __fp, int __type) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int __fsetlocking(FILE* _Nonnull __fp, int __type);
 
 __END_DECLS
