--- ./usr/include/bits/strcasecmp.h.orig	2023-12-14 19:41:52.021897736 -0500
+++ ./usr/include/bits/strcasecmp.h	2023-12-14 19:42:37.518761612 -0500
@@ -52,10 +52,7 @@
  * Like strcasecmp() but taking a `locale_t`.
  */
 
-#if __ANDROID_API__ >= 23
-int strcasecmp_l(const char* __s1, const char* __s2, locale_t __l) __attribute_pure__ __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int strcasecmp_l(const char* __s1, const char* __s2, locale_t __l) __attribute_pure__;
 
 /**
  * [strncasecmp(3)](http://man7.org/linux/man-pages/man3/strncasecmp.3.html) compares the first
@@ -71,9 +68,6 @@
  * Like strncasecmp() but taking a `locale_t`.
  */
 
-#if __ANDROID_API__ >= 23
-int strncasecmp_l(const char* __s1, const char* __s2, size_t __n, locale_t __l) __attribute_pure__ __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int strncasecmp_l(const char* __s1, const char* __s2, size_t __n, locale_t __l) __attribute_pure__;
 
 __END_DECLS
