--- ./usr/include/net/if.h.orig	2023-12-14 19:57:06.800268388 -0500
+++ ./usr/include/net/if.h	2023-12-14 19:57:39.058882109 -0500
@@ -47,11 +47,8 @@
 char* if_indextoname(unsigned __index, char* __buf);
 unsigned if_nametoindex(const char* __name);
 
-#if __ANDROID_API__ >= 24
-struct if_nameindex* if_nameindex(void) __INTRODUCED_IN(24);
-void if_freenameindex(struct if_nameindex* __ptr) __INTRODUCED_IN(24);
-#endif /* __ANDROID_API__ >= 24 */
-
+struct if_nameindex* if_nameindex(void);
+void if_freenameindex(struct if_nameindex* __ptr);
 
 __END_DECLS
 
