--- ./usr/include/pty.h.orig	2023-12-03 00:56:57.094273268 -0500
+++ ./usr/include/pty.h	2023-12-03 00:57:58.563461709 -0500
@@ -47,11 +47,9 @@
  *
  * Returns 0 on success and returns -1 and sets `errno` on failure.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int openpty(int* _Nonnull __pty_fd, int* _Nonnull __tty_fd, char* _Nullable __tty_name, const struct termios* _Nullable __termios_ptr, const struct winsize* _Nullable __winsize_ptr) __INTRODUCED_IN(23);
+int openpty(int* _Nonnull __pty_fd, int* _Nonnull __tty_fd, char* _Nullable __tty_name, const struct termios* _Nullable __termios_ptr, const struct winsize* _Nullable __winsize_ptr);
 
 /**
  * [forkpty(3)](http://man7.org/linux/man-pages/man3/forkpty.3.html) creates
@@ -60,10 +58,7 @@
  * Returns 0 in the child/the pid of the child in the parent on success,
  * and returns -1 and sets `errno` on failure.
  *
- * Available since API level 23.
  */
-int forkpty(int* _Nonnull __parent_pty_fd, char* _Nullable __child_tty_name, const struct termios* _Nullable __termios_ptr, const struct winsize* _Nullable __winsize_ptr) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int forkpty(int* _Nonnull __parent_pty_fd, char* _Nullable __child_tty_name, const struct termios* _Nullable __termios_ptr, const struct winsize* _Nullable __winsize_ptr);
 
 __END_DECLS
