--- ./usr/include/signal.h.orig	2023-10-21 19:14:37.562900732 +0700
+++ ./usr/include/signal.h	2023-10-21 19:16:05.159907076 +0700
@@ -179,10 +179,11 @@
 int pthread_sigmask64(int __how, const sigset64_t* __new_set, sigset64_t* __old_set) __INTRODUCED_IN(28);
 #endif /* __ANDROID_API__ >= 28 */
 
-
+#if __ANDROID_API__ >= 21
+int sigqueue(pid_t __pid, int __signal, const union sigval __value) __INTRODUCED_IN(21);
+#endif
 
 #if __ANDROID_API__ >= 23
-int sigqueue(pid_t __pid, int __signal, const union sigval __value) __INTRODUCED_IN(23);
 int sigtimedwait(const sigset_t* __set, siginfo_t* __info, const struct timespec* __timeout) __INTRODUCED_IN(23);
 #endif /* __ANDROID_API__ >= 23 */
 
