--- ./usr/include/sys/sysinfo.h.orig	2023-12-14 19:24:04.364631972 -0500
+++ ./usr/include/sys/sysinfo.h	2023-12-14 19:25:20.155070515 -0500
@@ -49,44 +49,33 @@
  * [get_nprocs_conf(3)](http://man7.org/linux/man-pages/man3/get_nprocs_conf.3.html) returns
  * the total number of processors in the system.
  *
- * Available since API level 23.
- *
  * See also sysconf().
  */
 
-#if __ANDROID_API__ >= 23
-int get_nprocs_conf(void) __INTRODUCED_IN(23);
+int get_nprocs_conf(void);
 
 /**
  * [get_nprocs(3)](http://man7.org/linux/man-pages/man3/get_nprocs.3.html) returns
  * the number of processors in the system that are currently on-line.
  *
- * Available since API level 23.
- *
  * See also sysconf().
  */
-int get_nprocs(void) __INTRODUCED_IN(23);
+int get_nprocs(void);
 
 /**
  * [get_phys_pages(3)](http://man7.org/linux/man-pages/man3/get_phys_pages.3.html) returns
  * the total number of physical pages in the system.
  *
- * Available since API level 23.
- *
  * See also sysconf().
  */
-long get_phys_pages(void) __INTRODUCED_IN(23);
+long get_phys_pages(void);
 
 /**
  * [get_avphys_pages(3)](http://man7.org/linux/man-pages/man3/get_avphys_pages.3.html) returns
  * the number of physical pages in the system that are currently available.
  *
- * Available since API level 23.
- *
  * See also sysconf().
  */
-long get_avphys_pages(void) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+long get_avphys_pages(void);
 
 __END_DECLS
