--- ./usr/include/sys/stat.h.orig	2023-12-14 19:37:40.128115592 -0500
+++ ./usr/include/sys/stat.h	2023-12-14 19:38:01.498521213 -0500
@@ -162,12 +162,7 @@
 // Implemented as a static inline before 21.
 #endif
 
-
-#if __ANDROID_API__ >= 23
-int mkfifoat(int __dir_fd, const char* __path, mode_t __mode) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
-
+int mkfifoat(int __dir_fd, const char* __path, mode_t __mode);
 int fchmodat(int __dir_fd, const char* __path, mode_t __mode, int __flags);
 int mkdirat(int __dir_fd, const char* __path, mode_t __mode);
 
