--- ./usr/include/utmp.h.orig	2021-08-16 11:52:08.095542840 +0200
+++ ./usr/include/utmp.h	2021-08-16 11:52:08.565542673 +0200
@@ -37,9 +37,9 @@
 #include <sys/types.h>
 #include <time.h>
 
-#define _PATH_UTMP      "/var/run/utmp"
-#define _PATH_WTMP      "/var/log/wtmp"
-#define _PATH_LASTLOG   "/var/log/lastlog"
+#define _PATH_UTMP      "@TERMUX_PREFIX@/var/run/utmp"
+#define _PATH_WTMP      "@TERMUX_PREFIX@/var/log/wtmp"
+#define _PATH_LASTLOG   "@TERMUX_PREFIX@/var/log/lastlog"
 
 #ifdef __LP64__
 #define UT_NAMESIZE 32
@@ -125,12 +125,8 @@
  *
  * Returns 0 on success and returns -1 and sets `errno` on failure.
  *
- * Available since API level 23.
  */
 
-#if __ANDROID_API__ >= 23
-int login_tty(int __fd) __INTRODUCED_IN(23);
-#endif /* __ANDROID_API__ >= 23 */
-
+int login_tty(int __fd);
 
 __END_DECLS
